README.md

```diff
@@ -7,21 +7,30 @@
 
 ## Latest
 
-We've been working on a lot of changes and ROCK 2.1 is here! Here's a quick look at some of the latest additions:
+We've been working on a lot of changes and ROCK 2.1 is here! You can read the full details in the [changelog](changelog.md), but here's a quick overview of some of the latest additions:
 
+#### 2.1 Changes
 - [x] Docket - click to PCAP utility
 - [x] official move to `.iso` distro model
-- [x] @todo
-- [x] @todo
+- [ ] @todo
```

Comment:
@seven62 check out the highlights here: https://github.com/rocknsm/rock/blob/master/RELEASE

Reply:
DONE